swag/blocks/swag_blocks.py

```diff
@@ -82,7 +83,7 @@ def execute(self, db: SwagChain):
 class Transaction(Block):
     giver_id = attrib(type=Union[UserId, CagnotteId])
     recipient_id = attrib(type=Union[UserId, CagnotteId])
-    amount = attrib(type=Union[Swag, Style])
+    amount = attrib(type=Union[Swag, Style, YfuId])
```

Comment:
I have very confused feelings about this

But I guess it is okay? I mean, I can't see anything wrong with that, and it technically makes the system simpler and more coherent

I need to get over it, sorry